pkg/webhook/config.go

```diff
@@ -88,6 +89,10 @@ func (cfg *Config) Validate() error {
 		return err
 	}
 
+	if strings.TrimSpace(cfg.RunnerLabel )== "" {
```

Comment:
Oops, my formatting mistake got applied beofore i could fix it
```suggestion
	if strings.TrimSpace(cfg.RunnerLabel) == "" {
```